Dockerfile

```diff
@@ -43,7 +43,14 @@ CMD ["nginx", "-g", "daemon off;"]
 
 # needed for chmod
 USER root
+
 # needed for write access for deployment.sh
 RUN mkdir /tmp/proxy_temp && chmod 0757 /usr/share/nginx/html/
+
```

Comment:
what is this for?

Reply:
There's a mismatch between the feature branch's base and the PR base. These changes are from acl2023_demo.